### ISSUE TITLE
Update wizard to remove reference to error color token

### ISSUE THREAD

**Issue by mcoker** [patternfly/patternfly-react#10335]
Follow up to https://github.com/patternfly/patternfly/pull/6447

TL;DR

#### Vertical nav/step list
* Adds new element `span.pf-v6-c-wizard__nav-link-status-icon` that is the first thing in `.pf-v6-c-wizard__nav-link` when there is an error on a step. 
  * Technically this element will hold any status icon, but so far we only support error status.
  * We're using the "exclamation circle" icon there since the step numbers are also circles. I think we usually use exclamation triangle, so calling that out.
* Adds `span.pf-v6-c-wizard__nav-link-main` that wraps everything else in `.pf-v6-c-wizard__nav-link` (`.pf-v6-c-wizard__nav-link-text` and `.pf-v6-c-wizard__nav-link-toggle`)
* When a step has an error, `.pf-v6-c-wizard__nav-link` getc class `.pf-m-danger`

#### Mobile nav toggle
* Adds new element `span.pf-v6-c-wizard__toggle-status-icon` that is the first thing in `.pf-v6-c-wizard__toggle-list-item` when the current step in the mobile nav toggle has an error
  * Same notes as the vertical nav notes above
* When the current step has an error, the `.pf-v6-c-wizard__toggle-list-item` in `.pf-v6-c-wizard__toggle` gets class `.pf-m-danger`

**Comment by mcoker** [comment]
cc @kmcfaul @tlabaj this is ready to work on and part of https://github.com/patternfly/patternfly-react/issues/10314

**Comment by kmcfaul** [comment]
Assigned this to myself and bringing this work in as part of the PR that will update the core version all together.
